airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/initialsync/MySqlInitialLoadHandler.java

```diff
@@ -49,7 +49,7 @@ public class MySqlInitialLoadHandler {
   private static final long RECORD_LOGGING_SAMPLE_RATE = 1_000_000;
   private final JsonNode config;
   private final JdbcDatabase database;
-  private final JdbcCompatibleSourceOperations sourceOperations;
+  private final MySqlSourceOperations sourceOperations;
```

Comment:
the implementation is always used, and implements a generic type. So we're getting rid of a rawtype warning